pkgs/development/python-modules/cirq/default.nix

```diff
@@ -27,50 +28,44 @@
 
 buildPythonPackage rec {
   pname = "cirq";
-  version = "0.9.1";
+  version = "0.10.0";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "quantumlib";
     repo = "cirq";
     rev = "v${version}";
-    sha256 = "0mygvpq7kzga8l1w2jvwv9a2n3akpss45hrx250gdrnqjp6xrw64";
+    sha256 = "0xinml44n2lfl0q2lb2apmn69gsszlwim83082f66vyk0gpwd4lr";
   };
 
   postPatch = ''
     substituteInPlace requirements.txt \
-      --replace "freezegun~=0.3.15" "freezegun" \
       --replace "matplotlib~=3.0" "matplotlib" \
       --replace "networkx~=2.4" "networkx" \
       --replace "numpy~=1.16" "numpy" \
-      --replace "protobuf~=3.12.0" "protobuf"
-
-    # Fix serialize_sympy_constants test by allowing small errors in pi
-    substituteInPlace cirq/google/arg_func_langs_test.py \
-      --replace "'float_value': float(str(np.float32(sympy.pi)))" "'float_value': pytest.approx(float(str(np.float32(sympy.pi))))"
+      --replace "protobuf~=3.13.0" "protobuf"
   '';
 
   propagatedBuildInputs = [
-    freezegun
     google-api-core
-    numpy
     matplotlib
     networkx
+    numpy
     pandas
     protobuf
     requests
     scipy
     sortedcontainers
     sympy
+    tqdm
     typing-extensions
   ];
 
-  doCheck = true;
   # pythonImportsCheck = [ "cirq" "cirq.Circuit" ];  # cirq's importlib hook doesn't work here
```

Comment:
Maybe that works now?

Reply:
Nope, just tried it now.

```
Check whether the following modules can be imported: cirq cirq.Circuit
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <lambda>
  File "/nix/store/5kznrncnp9v828a7i723j72bh9wkbkg7-python3-3.8.8/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/nix/store/kcfdn8fg28np26ch2d1qdkj8cf7lz57n-python3.8-cirq-0.10.0/lib/python3.8/site-packages/cirq/__init__.py", line 15, in <module>
    from cirq import _import
  File "/nix/store/kcfdn8fg28np26ch2d1qdkj8cf7lz57n-python3.8-cirq-0.10.0/lib/python3.8/site-packages/cirq/_import.py", line 25, in <module>
    class InstrumentedFinder(importlib.abc.MetaPathFinder):
AttributeError: module 'importlib' has no attribute 'abc'
```

Reply:
BTW, thank you for your many reviews of nixpkgs PRs! I've seen a lot of your reviews over the past few months, and the reviews posted with your ``nixpkgs-review-extensions`` (name??) seem really useful for catching the boring low-hanging-fruit review points.

Reply:
Its [nixpkgs-review-checks](https://github.com/SuperSandro2000/nixpkgs-review-checks) and thanks 😄